main.go

```diff
@@ -132,7 +113,7 @@ func main() {
 	}
 
 	templateVars["KEYCLOAK_URL"] = ""
```

Comment:
Why do we need `templateVars` ? 
In general, it isn't a good practice to use free form maps. You might need to wrap it in a `struct`

Reply:
not sure how to handle this!

Reply:
@sbose78 They are 'free' variables used to fill in the Openshift templates. `oc process -p`